modules/aws/rds_test.go

```diff
@@ -24,7 +24,7 @@ func TestGetRecommendedRdsInstanceTypeHappyPath(t *testing.T) {
 			databaseEngine:     "mysql",
 			engineMajorVersion: "8.0",
 			instanceTypes:      []string{"db.t2.micro", "db.t3.micro", "db.t3.small"},
-			expected:           "db.t2.micro",
+			expected:           "db.t3.micro",
```

Comment:
NIT: Consider jumping straight to `db.t4g.micro`, so you can delay dealing with this again for a while.